client/proxmark3.c

```diff
@@ -254,44 +369,35 @@ int main(int argc, char* argv[]) {
 		usb_present = true;
 		offline = 0;
 	}
-
-	// If the user passed the filename of the 'script' to execute, get it
-	if (argc > 2 && argv[2]) {
-		if (argv[2][0] == 'f' &&  //buzzy, if a word 'flush' passed, flush the output after every log entry.
-			argv[2][1] == 'l' &&
-			argv[2][2] == 'u' &&
-			argv[2][3] == 's' &&
-			argv[2][4] == 'h')
-		{
-			printf("Output will be flushed after every print.\n");
-			flushAfterWrite = 1;
-		}
-		else
-		script_cmds_file = argv[2];
+	
+	// dont execute comamnds in offline mode
+	if (offline && (executeCommand || script_cmds_file)) {
+		printf("ERROR: can't execute in offline mode.\n");
```

Comment:
Please remove. There are commands which can be executed in offline mode.

Reply:
deleted